### PR TITLE
Release azure spring data cosmos 3.0.0 beta.2

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Release azure spring data cosmos 3.0.0 beta.2

### CODE REVIEW

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
 ## Release History
 
-### 3.0.0-beta.2 (Unreleased)
-
+## 3.0.0-beta.2 (2020-09-17)
```
Comment (samvaity) [inline sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:3]: @weshaggard Could you confirm if this is the correct versioning strategy if the package has updated its group and artifact id.

image

**Review by samvaity — APPROVED** [review]

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:3]: @samvaity @weshaggard - group id is same -> `com.azure`, this is just an artifact id upgrade - 
from `azure-spring-data-2-3-cosmos` to `azure-spring-data-cosmos`

Comment (weshaggard) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:3]: We could reset the version but I think just incrementing it here is probably ok assuming we still plan to release this as a 3.0.0 GA.

**Review by saragluna — APPROVED** [review]
LGTM.

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md (hunk 1 above)
Comment (kushagraThapar) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:3]: @weshaggard - yes that is the plan - this is supposed to go GA as 3.0.0

**Pull request merged**